_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,83 @@
+
+For 60 years since the [1940s](https://en.wikipedia.org/wiki/Luria%E2%80%93Delbr%C3%BCck_experiment), all known bacterial immune processes could be boiled down to *innate* immunity: either you were born lucky with the right mutation, or you die. But in the mid-2000s, [several](https://pubmed.ncbi.nlm.nih.gov/15791728/) [groups](https://pubmed.ncbi.nlm.nih.gov/16079334/) [near](https://pubmed.ncbi.nlm.nih.gov/16545108/)-[simultaneously](https://pubmed.ncbi.nlm.nih.gov/15758212/) discovered something entirely new: an *adaptive* immune system that allows bacteria to gain new immunity to phages over the course of their short lifetimes. The section of bacterial DNA that contains this immune system had already been named CRISPR[^1] over a decade before, but now finally people knew what it did. 
+
+Bacteria with a CRISPR immune system can store a memory of past phage infections by cutting out small pieces of phage DNA and inserting these pieces in their own genome. The phage DNA mugshots are (unhelpfully) called *spacers*, and bacteria use them to recognize and bind to the DNA of phages that they've seen before, then neatly snip the phage DNA at that spot[^2], making the phage DNA useless and preventing the infection from proceeding. The important concepts to remember as we go on are that CRISPR immunity works through bacteria acquiring *spacers*, and that because spacers are stored in the genome, bacteria pass on their immunity to all their descendants as well.
+
+[^1]: CRISPR stands for Clustered Regularly Interspaced Short Palindromic Repeats, and it was given this name back in the 1990s by [Francisco Mojica](https://en.wikipedia.org/wiki/Francisco_Mojica) who first observed and characterized the genomic region that contains the CRISPR immune system in bacteria. It's a name that purely describes what the DNA looks like and says nothing at all about what it does!
+
+[^2]: This part of CRISPR immunity, the ability of CRISPR to cut DNA in a particular spot determined by what matches a spacer, has fueled a massive revolution in genome editing. The hard part of DNA editing is making an accurate cut, and up until the 2010s this was done by painstakingly engineering a particular protein for each edit. But when researchers Jennifer Doudna and Emmanuelle Charpentier were studying the CRISPR immune system, they realized that they could give the CRISPR-associated (Cas) proteins any DNA sequence they wanted instead of phage-derived spacers and that it would then make a precise DNA cut at the exact place they wanted. The pair received the [2020 Nobel Prize in Chemistry](https://www.nature.com/articles/d41586-020-02765-9) for their discovery.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/CRISPR_diagram_v3.svg" alt="crispr_diagram" width="600px" style="padding:5px;">
+</p>
+
+## Where do skewed spacer abundance distributions come from?
```

Comment:
Does this lead to a subtitle in the actual post? Otherwise the transition here can be made smoother.

Reply:
Yes, the ## gives a subheading, if that's what you mean.

Reply:
Yea that's what I meant. Looks good then!